tarun_kumar_PostgreAssignment/src/routes/routes.ts

```diff
@@ -0,0 +1,35 @@
+import Koa = require("koa");
+import koaRouter = require("koa-router");
+
+import { addClass, getClassId, getClasses } from '../controller/class-controller';
+import { addMarks, updateMarks } from '../controller/mark-controller';
+import { getStudents, addStudent, getStudentMarks } from '../controller/student-controller';
+import { getSubjects, addSubject } from '../controller/subject-controller';
+import { getTeachers, addTeacher } from '../controller/teacher-controller';
+
+
+const app = new Koa();
+export const router = new koaRouter();
+const port = 3001;
+
+//class
+router.get("/class", getClasses);
+router.get("/class/:name", getClassId);
+router.post('/class', addClass);
+
+//mark
+router.post('/mark', addMarks);
+router.put('/mark', updateMarks);
+
+//student
+router.get('/student', getStudents);
+router.post('/student', addStudent);
+router.get('/:student_id/marks', getStudentMarks);
+
```

Comment:
you have missed few api's like top 10 marks by class, top score for each subject/class.
Can you check once

Reply:
we need to get student given classId/subjectId/teacherId, for this we need schedule table(class, subject, and teacher - mapping).